python/tests/integration/apps/__init__.py

```
@@ -0,0 +1,34 @@
+_APPS_DIR = Path(__file__).resolve().parent
+
+
+def app_source(name: str) -> Path:
+    """Return the path to the app module ``name`` (without the ``.py`` suffix).
+
+    Pass the result as ``create_server_project(api_source=...)`` to install the
+    module as the subprocess project's ``api.py``.
+    """
+    path = _APPS_DIR / f"{name}.py"
+    if not path.exists():
+        raise FileNotFoundError(f"No integration test app module named {name!r} at {path}")
```

Ruff (0.15.17)

[warning] Avoid specifying long messages outside the exception class

(TRY003)